packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -136,6 +141,13 @@ contract DeployConfig is Script {
         preimageOracleMinProposalSize = stdJson.readUint(_json, "$.preimageOracleMinProposalSize");
         preimageOracleChallengePeriod = stdJson.readUint(_json, "$.preimageOracleChallengePeriod");
 
+        livenessModuleInterval = stdJson.readUint(_json, "$.livenessModuleInterval");
```

Comment:
Can you use the `_readOr` functions in this file to ensure that we don't break deploy configs that do not have these values set?

Reply:
Done, with a new _readOr for address arrays